XSUP-27024/exabeam (#29197)
demisto/content: Packs/Exabeam/ReleaseNotes/2_2_7.md

#### Integrations

##### Exabeam

- Updated the Docker image to: *demisto/python3:3.10.13.72123*.

> - Added a parameter *Minutes to look back when fetching*.

Reviewer: ```suggestion
- Added support for the *look-back* parameter in ***fetch-incidents***.
```
